### ISSUE TITLE
Set up target device 

### ISSUE THREAD

**Issue by fahim167** [SamuelTulach/mutante#11]
How to setup my computer as a target device ??

**Comment by atomik00** [comment]
?What you mean? 
no need for that just build a loader to load the driver or include it in the same project

**Comment by atomik00** [comment]
you don't run it from debug you have to load it via a Driver loader an example is osRloader or just write your own.

**Comment by fahim167** [comment]
I tried osrloader and it worked .. but it required to disable driver signature verification which easy anti cheat didn't like and refuse to start the game ..
is there away to load the driver without going into test mode ?? 
https://github.com/hfiref0x/TDL >> i found this loader but i can't figure out how to run it or it just won't 

i'm really thankful for your help ♥♥♥

**Comment by fahim167** [comment]
> you don't run it from debug you have to load it via a Driver loader an example is osRloader or just write your own.

I tried osrloader and it worked .. but it required to disable driver signature verification which easy anti cheat didn't like and refuse to start the game ..
is there away to load the driver without going into test mode ??
https://github.com/hfiref0x/TDL >> i found this loader but i can't figure out how to run it or it just won't

i'm really thankful for your help ♥♥♥

**Comment by atomik00** [comment]
Yeah you can bypass the need for disabling DSE, the bypass for that is Manual mapping your driver. look up tutorials on how to manual map a driver and once you get that you can map your driver to a vulnerable driver and voila your driver runs without issues or blue screens.
